beacon-chain/rpc/validator_server.go

```diff
@@ -226,3 +218,23 @@ func (vs *ValidatorServer) retrieveActiveValidator(beaconState *pbp2p.BeaconStat
 	}
 	return beaconState.ValidatorRegistry[validatorIdx], nil
 }
+
+func (vs *ValidatorServer) retrieveActiveValidatorsPublicKeys(pubkeys [][]byte) ([][]byte, error) {
+	m, err := vs.beaconDB.ValidatorIndices(pubkeys)
+	if err != nil {
+		return nil, fmt.Errorf("could not retrieve validators indexes: %v", err)
+	}
+	beaconState, err := vs.beaconDB.State(vs.ctx)
+	if err != nil {
+		return nil, fmt.Errorf("could not retrieve beacon state: %v", err)
+	}
+	activeIndices := helpers.ActiveValidatorIndices(beaconState.ValidatorRegistry, beaconState.Slot)
+	v := make([][]byte, 0, len(m))
```

Comment:
Shouldn't this be a map of pubkey -> activeIndex?

Reply:
you defined ```pb.ValidatorActivationResponse{
				ActivatedPublicKeys: activeKeys```
I can change it

Reply:
no, this is fine. thanks